schematic/models/metadata.py

```diff
@@ -235,3 +237,45 @@ def populateModelManifest(self, title, manifestPath: str, rootNode: str) -> str:
         emptyManifestURL = mg.get_manifest()
 
         return mg.populate_manifest_spreadsheet(manifestPath, emptyManifestURL)
+
+
+    def submit_metadata_manifest(self, manifest_path: str, dataset_id: str, validate_component: str, json_schema: str = None) -> bool:
+        """Wrap methods that are responsible for validation of manifests for a given component, and association of the 
```

Comment:
If validate_component is specified and the component exists in the schema, then do automatic JSON schema generation and validation. If the component doesn't exist then raise an exception.

Reply:
[L#268](https://github.com/Sage-Bionetworks/schematic/blob/develop-submit_metadata_manifest/schematic/models/metadata.py#L268) checks that `validate_component` exists in the schema. If not, it raises a `ValueError` exception.